platforms/gke/base/core/gke_enterprise/configmanagement/git/feature.tf

```diff
@@ -40,10 +43,20 @@ resource "google_gke_hub_feature_membership" "cluster_configmanagement" {
   feature    = google_gke_hub_feature.configmanagement.name
   location   = google_gke_hub_feature.configmanagement.location
   membership = data.google_container_cluster.cluster.name
-  project    = data.google_project.environment.project_id
+  project    = data.google_project.cluster.project_id
 
   configmanagement {
+    management = "MANAGEMENT_AUTOMATIC"
+
     config_sync {
```

Comment:
Can we also add the `prevent_drift` variable here?

https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/gke_hub_feature#prevent_drift-1

Thanks!